src/Payments.sol

```diff
@@ -35,13 +37,17 @@ interface IValidator {
 
 // @title Payments contract.
 contract Payments is ReentrancyGuard {
+    using Dutch for uint256;
     using SafeERC20 for IERC20;
     using RateChangeQueue for RateChangeQueue.Queue;
 
     // Maximum commission rate in basis points (100% = 10000 BPS)
     uint256 public constant COMMISSION_MAX_BPS = 10000;
+    uint88 private constant AUCTION_START_PRICE = 0.083 ether; // 0.083 FIL
```

Comment:
I'm sure I missed this in a doc somewhere but what is this based on as a start? I don't think we special-case USDFC in here anywhere so this is a generic start price for any ERC20, so how do we get to 0.083?

Reply:
it's in FIL. research picked this price. Maybe should be 4x higher to account for 1 week of decay